pkg/distributor/distributor.go

```diff
@@ -1189,10 +1189,13 @@ func (d *Distributor) UserStats(ctx context.Context) (*UserStats, error) {
 		totalStats.APIIngestionRate += r.ApiIngestionRate
 		totalStats.RuleIngestionRate += r.RuleIngestionRate
 		totalStats.NumSeries += r.NumSeries
+		totalStats.ActiveSeries += r.ActiveSeries
 	}
 
-	totalStats.IngestionRate /= float64(d.ingestersRing.ReplicationFactor())
-	totalStats.NumSeries /= uint64(d.ingestersRing.ReplicationFactor())
+	factor := d.ingestersRing.ReplicationFactor()
+	totalStats.IngestionRate /= float64(factor)
+	totalStats.NumSeries /= uint64(factor)
```

Comment:
So previously number of series is not divided by the factor? Do we want to change this in this pr?

Reply:
previously number of series is divided here, I just change `d.ingestersRing.ReplicationFactor()` to a uniform variable

Reply:
Make sense.